sofarpc-scenario/config/docker-composexx.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+services:
+  zk-server:
+    image: zookeeper:3.4.10
+    expose:
+      - "2181"
+    ports:
+      - "2181:2181"
+
+  sofarpc-consumer:
+    image: skywalking/sofarpc-consumer-scenario:5.3.2
+    expose:
+      - "8080"
+    ports:
+      - "8080:8080"
+    environment:
+      - COLLECTOR_SERVER=mock-collector:12800
+      - ZK_ADDRESS=zk-server:2181
+    links:
+      - zk-server
+      - sofarpc-provider
+      - mock-collector
+    volumes:
+      - /Users/bystander/Documents/code/open_code/incubator-skywalking/skywalking-agent:/usr/local/sofarpc-consumer/agent
+    depends_on:
+      mock-collector:
+        condition: service_healthy
+
+  sofarpc-provider:
+    image: skywalking/sofarpc-provider-scenario:5.3.2
+    environment:
+      - COLLECTOR_SERVER=mock-collector:12800
+      - ZK_ADDRESS=zk-server:2181
+    expose:
+      - "12200"
+    volumes:
+      - /Users/bystander/Documents/code/open_code/incubator-skywalking/skywalking-agent:/usr/local/sofarpc-provider/agent
```

Comment:
The AgentPath should be `{AGENT_FILE_PATH}` instead of `/Users/bystander/Documents/code/open_code/incubator-skywalking/skywalking-agent`

Reply:
@ascrutae this file will not be used ,I pushed wrong file.